File: 00921.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
	int n;
	cin >> n;
	vector<float> nums(n);
	
	for(int i = 0; i < n; i++){
		cin >> nums[i];	
	}
	float sum = 0;
	int count = 0;
	for(int i = 0; i < n; i++){
		if(nums[i] < 0){
			sum = sum + nums[i];
			count++;
		}
		
	}
	cout << count << " " << fixed << setprecision(2) << sum;
}

File: 00921.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        double nums[] = new double[n];
        int count = 0;
        double sum = 0;

        for(int i = 0; i < n; i++){
            nums[i] = scan.nextDouble();
            try{
                if(nums[i] < 0){
                    count++;
                    sum = sum + nums[i];
                }
            }
            catch(Exception e){
                scan.next();
                i--;
            }
        }
        System.out.print(count + " " + String.format("%.2f", sum));
    }
}
